persistence-utils-test/pom.xml

```diff
@@ -37,47 +37,12 @@
 		<version>3.7.0</version>
 		<scope>test</scope>
 	</dependency>
-	<dependency>
-		<groupId>org.hamcrest</groupId>
-		<artifactId>hamcrest</artifactId>
-		<version>2.2</version>
-		<scope>test</scope>
-	</dependency>
 	<dependency>
 		<groupId>org.hsqldb</groupId>
 		<artifactId>hsqldb</artifactId>
 		<version>2.5.0</version>
 		<scope>test</scope>
 	</dependency>
-	<dependency>
```

Comment:
so good to see powermock giving way to simple maintainable @IvyTest driven solutions